sormas-base/setup/import-to-truststore.sh

```diff
@@ -0,0 +1,104 @@
+
+#*******************************************************************************
+# SORMAS® - Surveillance Outbreak Response Management & Analysis System
+# Copyright © 2016-2018 Helmholtz-Zentrum f�r Infektionsforschung GmbH (HZI)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#*******************************************************************************
+
+#!/bin/bash
+
+echo "# SORMAS TO SORMAS CERTIFICATE IMPORT"
+echo "# This script imports a certificate into the local truststore, to be used for SORMAS2SORMAS and SurvNet communication"
+echo "# If anything goes wrong, please consult the sormas to sormas import guide or get in touch with the developers."
+
+if [[ $(expr substr "$(uname -a)" 1 5) = "Linux" ]]; then
+	LINUX=true
+else
+	LINUX=false
+fi
+
+# DIRECTORIES
+if [[ ${LINUX} = true ]]; then
+	ROOT_PREFIX=
+else
+	ROOT_PREFIX=/c
+fi
+if [[ -z "${SORMAS2SORMAS_DIR}" ]] || [[ ! -d "${SORMAS2SORMAS_DIR}" ]]; then
+  DEFAULT_SORMAS2SORMAS_DIR="${ROOT_PREFIX}/opt/sormas2sormas"
```

Comment:
If a SORMAS2SORMAS_DIR is specified, but does not exist, the script should fail.